.github/templates/odfe-testing-cluster-static-domain-template.json

```diff
@@ -36,21 +36,25 @@
 
         "esTargetGroup":{
             "Type" : "String",
-            "Description": "ES Target Group ARN"
+            "Description": "ES Target Group ARN with port 9200"
         },
         
         "kibanaTargetGroup":{
             "Type" : "String",
-            "Description": "Kibana Target Group ARN"
+            "Description": "Kibana Target Group ARN with port 5601"
+        },
+        "paTargetGroup":{
+            "Type" : "String",
+            "Description": "PA Target Group ARN with port 9600"
         }
   
     },
 
     "Mappings" : {
         "DistributionMap" : {
-          "RPM" : { "amiId" : "ami-0e34e7b9ca0ace12d" },
-          "TAR" : { "amiId" : "ami-003634241a8fcdec0" },
-          "DEB" : { "amiId" : "ami-003634241a8fcdec0" }
+          "RPM" : { "amiId" : "ami-01ce4793a2f45922e" },
+          "TAR" : { "amiId" : "ami-07a29e5e945228fa1" },
+          "DEB" : { "amiId" : "ami-07a29e5e945228fa1" }
```

Comment:
What is the difference between these AMIs? It is surprising that tar and deb share an AMI, but rpm does not.

Adding a comment explaining the difference would be super helpful.

Reply:
RPM- Amazon Linux Machine 
DEB/RPM - Ubuntu
I wish I can add comments but its JSON :(

Reply:
Not surprising at all, TAR just need a generic linux so sharing the same ubuntu image is ok.
As for the changes, @gaiksaya was initially using AMI provided by community, which has /dev/shm as 755, thus cause issues with PA writing to the path with non-root user. We changed it to the official Ubuntu/AL2 images provided by AWS, which have 777 for /dev/shm by default here.

Reply:
Oh yeah, JSON :(

Thanks for the explanation.